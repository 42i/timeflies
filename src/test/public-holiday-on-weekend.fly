day 2012-05-03 9 18.5, off 1.5
; hilarious hoovering
day 2012-05-04, phol; Frugal Friday
day 2012-05-05, phol; Mad Hatters' Day
day 2012-05-06, phol; Seagoon Sunday
day 2012-05-07 8 17, off 1
; dreadful drudgery
day 2012-05-08
must-hours wed=4
day 2012-05-08, phol; Blathering Bluesday
day 2012-05-09, phol; Harking Hayday
