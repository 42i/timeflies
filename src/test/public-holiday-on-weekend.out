Time at work overview (all):
     when        worked   leave    sick balance
2012-05-03 Thu:    8.00 ----.-- ----.-- ----.--
2012-05-04 Fri: ----.-- ----.-- ----.-- ----.-- Frugal Friday
  week 2012-18:    8.00 ----.-- ----.-- ----.--
2012-05-07 Mon:    8.00 ----.-- ----.-- ----.--
2012-05-09 Wed: ----.-- ----.-- ----.-- ----.-- Harking Hayday
  week 2012-19:    8.00 ----.-- ----.-- ----.--
 month 2012-05:   16.00 ----.-- ----.-- ----.--
         total:   16.00 ----.-- ----.-- ----.--
     when        worked   leave    sick balance
